std-bits/google/src/main/java/org/enso/google/GoogleSheetsForEnso.java

```diff
@@ -41,7 +41,7 @@ public static GoogleSheetsForEnso create(WrappedGoogleCredentials credentials)
     return new GoogleSheetsForEnso(builder.build());
   }
 
-  public Table getSheetRange(
+  public Object getSheetRange(
```

Comment:
- need a common super type of `Table` and `Value` because of [returning Error.throw](https://github.com/enso-org/enso/pull/14040#discussion_r2413027910)

Reply:
Alternatively if we inline the Panic.catch then it will catch the correct exception.